data/strings/types_strings.txt

```diff
@@ -26951,7 +26954,7 @@
   [type.piste_type.downhill.advanced]
     ref = type.piste_type.downhill
     en = Advanced Downhill Ski Run
-    de = Schwierige Abfahrt
+    de = Schwierige Ski Abfahrt
```

Comment:
Ski-Abfahrt

Reply:
No, it is important to distinguish the complexity. Users can check it by a long-tap.

Reply:
You got me wrong. I wanted to say that the hyphen is missing. Sorry for not beeing clear. In this exact case it should say:
Schwierige Ski-Abfahrt
Without hyphen it is grammatically wrong.

Reply:
Thank you for the clarification!

Reply:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Schwierige Ski-Abfahrt
```